### PR TITLE
Add monitoring and alerting for oss-prow

### PR DESCRIPTION
**Pull request by chaodaiG**
This PR consists of 2 commits:

- Blindly copy/paste from k8s https://github.com/kubernetes/test-infra/tree/master/config/prow/cluster/monitoring
- Update based on oss-prow

The dashboard is only visible by port forwarding, as the ingress isn't configured yet

### CODE REVIEW

File: mixins/dashboards_out/boskos-http.json [hunk 1]
```diff
@@ -0,0 +1,304 @@
+{
```
Comment (cjwagner) [inline mixins/dashboards_out/boskos-http.json:1]: ~Please run `make clean`.~ (This is a top level `mixins` dir not `prow/cluster/monitoring/mixins` so we just need to delete it.) We don't want to check in the generated dashboards and alerts.

Comment (cjwagner) [reply mixins/dashboards_out/boskos-http.json:1]: Also why are there dashboards for boskos? Those should have been removed. It looks like they were mistakenly deployed to the cluster as well so we'll need to clean those up, but we should figure out what went wrong to allow that first.

File: prow/oss/cluster/monitoring/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+* For `prometheus` and `alertmanager`, there is no public domain configured based on the security
+concerns (no authorization out of the box).
+Cluster admins can use [k8s port-forward](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/) to
+access the web.
+
+    ```
+    $ kubectl -n prow-monitoring port-forward $( kubectl -n prow-monitoring get pods --selector app=prometheus -o jsonpath={.items[0].metadata.name} ) 9090
+    $ kubectl -n prow-monitoring port-forward $( kubectl -n prow-monitoring get pods --selector app=alertmanager -o jsonpath={.items[0].metadata.name} ) 9093
+    ```
```
Comment (cjwagner) [inline prow/oss/cluster/monitoring/README.md:78]: Add this for grafana
```
kubectl -n prow-monitoring port-forward $( kubectl -n prow-monitoring get pods --selector app=grafana -o jsonpath={.items[0].metadata.name} ) 8080:80
```

File: prow/oss/cluster/monitoring/additional-scrape-configs_secret.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+stringData:
+  prometheus-additional.yaml: |
+    - job_name: blackbox
+      metrics_path: /probe
+      params:
+        module: [http_2xx]
+      static_configs:
+        - targets:
+          # ATTENTION: Keep this in sync with the list in mixins/prometheus/lib/config.libsonnet
+          - https://oss-prow.knative.dev/monitoring
```
Comment (cjwagner) [inline prow/oss/cluster/monitoring/additional-scrape-configs_secret.yaml:15]: This is not in sync.

**Review by cjwagner — CHANGES_REQUESTED** [review]

**Comment by cjwagner** [comment]
We can probably just delete the vendored OWNERS files.

File: mixins/dashboards_out/boskos-http.json (hunk 1 above)
Comment (chaodaiG) [reply mixins/dashboards_out/boskos-http.json]: Bad copy/paste made these files here, there is another copy at the correct path

File: prow/oss/cluster/monitoring/additional-scrape-configs_secret.yaml (hunk 3 above)
Comment (chaodaiG) [reply prow/oss/cluster/monitoring/additional-scrape-configs_secret.yaml]: fixed

File: prow/oss/cluster/monitoring/README.md (hunk 2 above)
Comment (chaodaiG) [reply prow/oss/cluster/monitoring/README.md:79]: Added

**Comment by chaodaiG** [comment]
> We can probably just delete the vendored OWNERS files.

Deleted, addressed other comments as well

**Review by cjwagner — APPROVED** [review]

**Pull request merged**